tests/unit/test-util/test.temp-dir.js

```diff
@@ -58,4 +58,9 @@ describe('util.TempDir', () => {
     assert.throws(() => tmp.path(), /cannot access path.* before.* create/);
   });
 
+  it('does not throw on remove called before a temp dir is created', () => {
+    const tmp = new TempDir();
+    assert.equal(tmp._removeTempDir, undefined);
```

Comment:
If you're testing this internal, also add a test where `create` is called, and check that the property exist. That will ensure that the test gets the desired attention if the `TempDir` class ever gets refactored.

Reply:
Main reason for testing that internal was avoiding coverall failure :-)
But given that we are testing it, let's also add the additional assertion ;-)

done in the updated version of this PR.